server/public/index.go.html

```diff
@@ -6,17 +6,45 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>{{ __ "offen" }}</title>
   <link rel="stylesheet" type="text/css" href="/normalize.css">
-  <link rel="stylesheet" type="text/css" href="/skeleton.css">
+  <link rel="stylesheet" type="text/css" href="/lit.css">
+  <link rel="stylesheet" type="text/css" href="/fix.css">
   {{ with .rootAccount }}
     <script src="/script.js" data-account-id="{{ . }}"></script>
   {{ end }}
 </head>
 <body>
-  <div class="container">
+  <div class="c">
     <div class="row">
-      <div class="twelve columns">
-        <h1>{{ __ "Welcome to offen" }}</h1>
-        <p>{{ __ "Access your data in the <a href=\"auditorium\">auditorium</a>" }}</p>
+      <h1>{{ __ "Welcome to <strong>offen</strong>" }}</h1>
+    </div>
+    <div class="row"></div>
+    <div class="row">
+      <div class="card">
+          <div class="row">
+            <div class="2 col"></div>
+            <div class="8 col">
+              <div class="row"></div>
+              <div class="row">
+                <form action="auditorium" method="get">
```

Comment:
This is not a form as it is not sending any data. The button should be an anchor element pointing to the desired href instead.

Reply:
I used this only for the looks - as it seems not possible to do full width buttons with basic lit - Ill try the utility package